src/components/detailedView/instanceView/instanceView.tsx

```diff
@@ -2,106 +2,75 @@ import { Classes } from "@blueprintjs/core";
 import "@blueprintjs/core/lib/css/blueprint.css";
 import axios from "axios";
 import * as React from "react";
+import { API_ROOT } from "../../../api-config";
+import PropertiesView from "../propertiesView";
 import { RouteComponentProps, withRouter } from "react-router";
 import "./instanceView.scss";
+import { connect } from "react-redux"
+import { InstanceObject } from "../../utils"
 
 export interface InstanceViewProps {
-  rid: string;
+  token: string;
+  rid: any;
 }
-
 // Given an rid, will perform a GET request of that rid and display info about that instnace
 
-async function getData(instancekey: string) {
+async function getData(instancekey: string, token: string) {
+  const headers = {
+    headers: {
+      Authorization: "Token " + token
+    }
+  }
   return await axios
-    .get("https://rack-city-dev.herokuapp.com/api/instances/" + instancekey)
+    .get(API_ROOT + "api/instances/" + instancekey, headers)
     .then(res => {
       const data = res.data;
       return data;
     });
 }
 
-export class InstanceViewWrapper extends React.PureComponent<
-  RouteComponentProps,
-  InstanceViewProps
-> {
-  public render() {
-    return (
-      <InstanceView
-        history={this.props.history}
-        location={this.props.location}
-        match={this.props.match}
-      />
-    );
-  }
-}
-
 interface InstanceViewState {
-  hostname: any;
-  model: any;
-  rack: any;
-  height: any;
-  user: any;
-  comment: any;
+  state: InstanceObject | undefined,
```

Comment:
thank u for refactoring this!! I'm going to use this InstanceObject to populate data for the modify instance form, so this will make connecting that in a lot easier.